edx_django_utils/monitoring/scripts/run_nrql_on_nerd_graph.py

```diff
@@ -0,0 +1,36 @@
+import requests
+import json
+import os
+
+
+def run_nrql_query(query, api_key, account_id):
+    """
+    Use to run nrql query on new relic data.
+
+    The function uses new_relic's graphql api.
+    More details on this can be found at: https://developer.newrelic.com/collect-data/get-started-nerdgraph-api-explorer/"""
+    graphql_url = 'https://api.newrelic.com/graphql'
+    headers = {'Content-Type': 'application/json', 'API-Key': api_key}
+    graphql_query = f"""{{
+   actor {{
+      account(id: {account_id}) {{
+         nrql(query: "{query}") {{
+            results
+         }}
+      }}
+   }}
+}}"""
+    post_data = {'query': graphql_query.replace('\n','')}
+    response = requests.post(url=graphql_url,  headers = headers, json=post_data)
```

Comment:
Might results be paged like for other calls? If so, should we add a comment/warning that this script doesn't handle paging? See example here: https://github.com/edx/edx-django-utils/blob/master/edx_django_utils/monitoring/scripts/new_relic_nrql_search.py#L166-L167